services/component-repository/package-lock.json

```diff
@@ -0,0 +1,10185 @@
+{
```

Comment:
There shouldn't be any package-lock json file in the services or libraries, just in the root level